lib/active_admin/views/pages/show.rb

```diff
@@ -25,7 +25,11 @@ def main_content
         end
 
         def attributes_table(*args, &block)
+          opts = args.extract_options!
           table_title = ActiveAdmin::Localizers.resource(active_admin_config).t(:details)
```

Comment:
I think ActiveAdmin::Localizers.resource(active_admin_config).t(:details) should work as fallback if ```: title``` options was not passed